vendor/engines/projects/app/views/projects/projects/edit.html.haml

```diff
@@ -0,0 +1,15 @@
+= render "shared/headers/ckeditor"
+
+= content_for :h1 do
+  = @project.facility
+
+%h2= t(".update")
+
+= simple_form_for @project, url: facility_project_path(@project.facility, @project), method: :put do |f|
```

Comment:
It's close; `edit` has the `:active` checkbox and `new` doesn't. I lean toward keeping them separate for now.